### DISCUSSION TITLE
Using the expanded_form for larger spins

### DISCUSSION THREAD

**Discussion by thefacemyer** [QuSpin/QuSpin#663] [Q&A]
I am trying to apply a transverse field to a spin system that has spin > 1/2. I see that I need to change my "x" to the "+/-" combination using the expanded_form function. I'm sure I'm overlooking it in the documentation, but I cannot find what library I need to import from. Can someone help? And, do I simply call the static part as hamiltonian(static=expanded_form[0], dynamic=expanded_form[1], ...)?

**Comment by mgbukov** [comment]
I don't think you need an extra library; just set say S='1' in the basis
constructor and in the static/dynamic lists, express the Hamiltonian using
the +/- operators (there may be extra factors of 1/2 you need to ass to.the
coupling strength); for 'y' they are also conplex-valued. You can do this
pen-on-paper first and then code up the expressions using the +/- opstr's.

**Reply by thefacemyer** [reply]
Yes, that's what I've done in the meantime. Is there not a computational advantage to using the function? If not, then I'll stick with your suggestion. Thanks!

**Comment by mgbukov** [comment]
I think this function is used internally when checking for symmetry
operations; I'm not sure if there's that big of an advantage for simple
models like TFIM.
